### DISCUSSION TITLE
Simulating VcMediaNext also sends P key

### DISCUSSION THREAD

**Discussion by BlackCoyote** [TolikPylypchuk/SharpHook#80] [Q&A] [answered]
Performing the following line of code on windows (10):

`new EventSimulator().SimulateKeyPress(SharpHook.Native.KeyCode.VcMediaNext);`

also seems to simulate the P key.  I used this website to test simulated keys: https://en.key-test.ru/, I also experienced this in other applications that react to the P key.
I don't experience the same behaviour when I simply press the MediaNext key on my keyboard.

I did also experience this when I tried a different library for simulating keystrokes, so I don't expect this to be an issue with the library, but I was really hoping I could find some help here.

**Comment by TolikPylypchuk** [comment]
Hello! Thanks for posting this discussion! This definitely sounds weird and I will investigate it.

**Comment by TolikPylypchuk** [comment, marked as answer]
I've created issue #81 based on this discussion, so I will close the discussion.

**Reply by BlackCoyote** [reply]
Hey,

Thanks for the quick reply and to keep an issue open for this. Really refreshing to see someone take such care and interest!

I'll do my best to share any findings that might be relevant to the issue. Feel free to reach out if you'd like me to test something too.
